packages/core/src/button/Button.css

```diff
@@ -54,7 +54,7 @@
 
 /* Icon API in "primary" variant @ignore */
 .uitkButton-primary {
-  --uitkIcon-color: var(--uitk-actionable-primary-foreground);
+  --uitkIcon-color: var(--uitkButton-iconColor, var(--uitk-actionable-primary-foreground));
```

Comment:
I was thinking to keep camel case to just CSS attributes and component names. since icon-color isn't a css attribute, as well as text-color, i was going to keep those kebab case. thoughts? (I just put up a FormField PR where you can see where I have changed to camel and where not) happy to discuss